prebackup/prebackup.go

```diff
@@ -34,14 +36,38 @@ func NewPrebackup(config job.Config) *PreBackup {
 	}
 }
 
+const (
+	// ConditionPreBackupPodsAvailable is True if there are Container definitions
+	ConditionPreBackupPodsAvailable status.ConditionType = "PreBackupPodsAvailable"
+
+	// ConditionPreBackupPodsReady is True if Deployments for all Container definitions were created and are ready
+	ConditionPreBackupPodsReady status.ConditionType = "PreBackupPodsReady"
+
+	// ConditionPreBackupPodsRemoved is True if Deployments for all Container definitions were created
+	ConditionPreBackupPodsRemoved status.ConditionType = "PreBackupPodsRemoved"
+)
+
 // Start will start the defined pods as deployments.
 func (p *PreBackup) Start() error {
 
+	p.SetConditionsUnknown(
+		ConditionPreBackupPodsAvailable,
+		ConditionPreBackupPodsReady,
+	)
```

Comment:
(applies to other job types as well)
I would advise to only set conditions once they're actually needed for the first time. Setting all possible conditions to unknown right at the start may confuse the users reading them (it's so easy to read a condition and assume they're true by overlooking the "unknown" property).